### DISCUSSION TITLE
Training Tortoise

### DISCUSSION THREAD

**Discussion by OccultMC** [neonbjb/tortoise-tts#157] [Q&A]
Hi, Betker!
Is there any documentation on training Tortoise on a dataset of a particular voice?
Thanks

**Comment by neonbjb** [comment]
Hey there,
I have intentionally withheld the capability to fine-tune Tortoise because of safety concerns. I apologize for this, but I believe it is the right thing to do.

**Reply by vdruts** [reply]
First off thank you for your work. Second, I truly hope you will reconsider opening this up and allowing us to train. As you have mentioned corporations are already doing this and the only true way to ensure we can all benefit from this is to open it up much like Stability.ai. Yes this may create issues, but technology always does. The fight of good versus evil is solely at the discretion of individuals. Fire can be used to create food and warmth or to burn and mame. 

I am excited about all the amazing things that can be done with training.

Thanks for considering and thanks again for your work :)

**Reply by ghost** [reply]
+1
